stubs/2.7/builtins.pyi

```diff
@@ -37,7 +37,10 @@ class type:
     __module__ = ''
     __dict__ = ...  # type: Dict[unicode, Any]
 
+    @overload
     def __init__(self, o: object) -> None: ...
+    @overload
+    def __init__(self, name: str, bases: tuple, dict: Dict[str, Any]) -> None: ...
```

Comment:
But until we can have both, `Sequence[type]` will allow a stronger subset of the desired type-checking.